### PR TITLE
[PO-42] Simple mining startup

### PR DESCRIPTION
**Pull request by mirkoAlic**
The goal of this PR is to add a simple way of unlock "mining" accounts in order to allow them to sign blocks.

`unlockMinerAccount.sh`
```
#!/bin/bash

server_and_port=$1 # http://127.0.0.1:8546
address=$2 # 0xedde8656c35fcb7126c61fc6e2673734425a72bf
password=$3 # 1234

echo $server_and_port $address $password

curl -X POST \
  "$server_and_port"/ \
  -H 'cache-control: no-cache' \
  -H 'content-type: application/json' \
  -d '{ "jsonrpc": "2.0", "method": "personal_unlockMinerAccount", "params": ["'$address'", "'$password'"], "id": 1}'
```

example of use:
```
./unlockMineAccount.sh http://127.0.0.1:8546 "0xedde8656c35fcb7126c61fc6e2673734425a72bf" "1234"
```

### CODE REVIEW

File: src/scripts/unlockMineAccount.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (ntallar) [inline src/scripts/unlockMineAccount.sh]: Minor detail: I'm not sure if this script is necessary, maybe we should add it to the PR description?

**Review by ntallar — APPROVED** [review]
Apart from a minor detail, LGTM!

**Comment by mirkoAlic** [comment]
new PR target to: https://github.com/ntallar/mantis/pull/14

**Comment by mirkoAlic** [comment]
new PR target to: https://github.com/ntallar/mantis/pull/14

**Pull request closed without merging**